packages/contract/src/pay/DaimoPayRelayer.sol

```diff
@@ -91,30 +130,34 @@ contract DaimoPayRelayer is Ownable2Step {
             address(this)
         ) - amountPreSwap;
 
-        // Check the amount output from the swap. The contract owner tips the
-        // difference if needed. If there are excess tokens, transfer them to
-        // the owner.
+        //////////////////////////////////////////////////////////////
+        // POST-SWAP
+        //////////////////////////////////////////////////////////////
+
+        // If we received less than required, check that the amount we need to
+        // tip is within maxPostTip.
         if (swapAmountOut < requiredTokenOut.amount) {
-            // Output more tokens from owner.
             uint256 outShortfall = requiredTokenOut.amount - swapAmountOut;
-            require(outShortfall <= maxTip, "DPR: excessive tip");
-            TokenUtils.transferFrom(
-                requiredTokenOut.token,
-                owner(),
-                address(this),
-                outShortfall
-            );
-        } else {
-            // Give excess tokens to owner.
-            uint256 tip = swapAmountOut - requiredTokenOut.amount;
-            TokenUtils.transfer(requiredTokenOut.token, payable(owner()), tip);
+            require(outShortfall <= maxPostTip, "DPR: excessive post tip");
         }
 
+        // Transfer the required output tokens to the caller, tipping the
+        // shortfall if needed. If there are surplus tokens from the swap, keep
+        // them.
```

Comment:
one effect of this is that the relayer will accumulate change in whatever the destination tokens are.

for ethOS, for example, it'll accumulate ETH on Base.

Reply:
yes